core/swift3Action/spm-build/_Whisk.swift

```diff
@@ -24,27 +24,40 @@ class Whisk {
         let strBlocking = blocking ? "true" : "false"
         let path = "/api/v1/namespaces/\(parsedAction.namespace)/actions/\(parsedAction.name)?blocking=\(strBlocking)"
 
-        return postSyncronish(uriPath: path, params: params)
+        return sendWhiskRequestSyncronish(uriPath: path, params: params, method: "post")
     }
 
     class func trigger(eventNamed event : String, withParameters params : [String:Any]) -> [String:Any] {
         let parsedEvent = parseQualifiedName(name: event)
         let path = "/api/v1/namespaces/\(parsedEvent.namespace)/triggers/\(parsedEvent.name)?blocking=true"
 
-        return postSyncronish(uriPath: path, params: params)
+        return sendWhiskRequestSyncronish(uriPath: path, params: params, method: "post")
+    }
+
+    class func createTrigger(triggerNamed trigger: String, withParameters params : [String:Any]) -> [String:Any] {
```

Comment:
Unfortunately I don't happen know the proper dance to make this happen correctly. But the CLI does: https://github.com/openwhisk/openwhisk-cli/blob/master/commands/trigger.go#L103

In any case, I don't think we should hold up this PR just because that feature is missing.